cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,635 @@
+        nav_menu = VersionPicker(
+            {
+                Version.lowest(): ["Compute", "Migration"],
+                "5.10": ["Compute", "Migration", "Migration Plans"],
+            }
+        ).pick()
+        return self.logged_in_as_current_user and self.navigation.currently_selected == nav_menu
+
+    @property
+    def is_displayed(self):
+        return self.in_explorer
+
+
+class DashboardCards(View):
+    not_started_plans = MigrationDashboardStatusCard("Not Started")
+    in_progress_plans = MigrationDashboardStatusCard("In Progress")
+    completed_plans = MigrationDashboardStatusCard("Complete")
+    archived_plans = VersionPicker(
+        {
+            Version.lowest(): MigrationDashboardStatusCard("Infrastructure Mappings"),
+            "5.10": MigrationDashboardStatusCard("Archived"),
+        }
+    )
+
+    def read(self):
+        return [c.read().get("name") for c in self.sub_widgets if c.is_active]
+
+
+class MigrationPlanView(MigrationView):
+    dashboard_cards = View.nested(DashboardCards)
+    paginator_view = View.include(V2VPaginatorPane, use_parent=True)
+    title = Text('.//div[contains(@class, "pull-left")]//h3')
+    create_migration_plan = Text(locator='(//a|//button)[text()="Create Migration Plan"]')
+    configure_providers = Text(locator='//a[text()="Configure Providers"]')
+    sort_type_dropdown = SelectorDropdown("id", "sortTypeMenu")
+    sort_direction = Text(locator=".//span[contains(@class,'sort-direction')]")
+    progress_card = MigrationProgressBar()
+    search_box = TextInput(locator=".//div[contains(@class,'input-group')]/input")
+    clear_filters = Text(".//a[text()='Clear All Filters']")
+    # 5.9 field
+    migr_dropdown = MigrationDropdown(text="Not Started Plans")
+
+    @property
+    def in_migration_plan(self):
+        return (
+            self.in_explorer and self.create_migration_plan.is_displayed and
+            len(self.browser.elements('.//div[contains(@class,"card-pf")]')) > 0
+            and len(self.browser.elements(PFIcon.icons.WARNING)) < 1
+        )
+
+    @property
+    def is_displayed(self):
+        return self.in_migration_plan
+
+
+class NotStartedPlansView(MigrationPlanView):
+    plans_not_started_list = MigrationPlansList("plans-not-started-list")
+
+    @property
+    def is_displayed(self):
+        expected_title = 'Not Started Plans'
+        on_plan_page = self.in_migration_plan and self.plans_not_started_list.is_displayed
+
+        if self.context['object'].appliance.version >= '5.10':
+            return on_plan_page and self.title.text == expected_title
+        else:
+            return on_plan_page
+
+
+class InProgressPlansView(MigrationPlanView):
+    progress_card = MigrationProgressBar()
+
+    @property
+    def is_displayed(self):
+        expected_title = 'In Progress Plans'
+        on_in_progress_page = self.in_migration_plan and self.progress_card.is_displayed
+
+        if self.context['object'].appliance.version >= '5.10':
```

Comment:
Since the 5.9 breakpoint has been set, we can remove this check here and the other `is_displayed` methods